docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-editor.scss

```diff
@@ -9,3 +9,23 @@ li.cke_panel_listItem a:focus {
 body.cke_editable {
   margin: hb-calculate-rems(8px);
 }
+
+// Makes sure that all images in the WYSIWYG editor are a level above the other
+// content and can be selected to edit.
+/* stylelint-disable */
+drupal-media {
+  position: relative;
+  z-index: $hb-z-index-figure;
+
+  figure,
```

Comment:
nice workaround! so you didn't end up needing your JS to go here right? you're just using styles instead?

Reply:
Yeah, it isn't a perfect solution since it affects images with captions but nothing looks broken, there is just some extra spacing around images. This should be fine since the WYSIWYG preview isn't exactly what they get on the frontend, just an approximation. 

The su_humsci theme did not address this in the editors, videos with captions had a width of 0px so I figured this fix is better than nothing.